src/MongoDB.Driver.Core/Core/Configuration/ConnectionString.cs

```diff
@@ -756,7 +756,7 @@ private void Parse()
             const string optionsPattern = @"(\?" + optionPattern + @"((&|;)" + optionPattern + ")*)?";
             const string pattern =
                 @"^(?<scheme>mongodb|mongodb\+srv)://" +
-                @"((?<username>[^:@]+)(:(?<password>[^:@]*))?@)?" +
+                @"((?<username>[^:@/]+)(:(?<password>[^:@/]*))?@)?" +
```

Comment:
I presume this change was necessary to handle [this test](https://github.com/mongodb/specifications/commit/4631ccd4f825fb1a3aba204510023f9b4d193a05#diff-84c3634193c53eae6131bd8cf7b788d6R196) that was made without a spec ticket?

Reply:
Yup!